test/kpod-mount.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+function teardown() {
+    cleanup_test
+}
+
+load helpers
+
+IMAGE="alpine:latest"
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
+
+@test "mount" {
+    start_crio
+    run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    pod_id="$output"
+    run crioctl ctr create --config "$TESTDATA"/container_config.json --pod "$pod_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr_id="$output"
+    stop_crio
+    root=$(${KPOD_BINARY} ${KPOD_OPTIONS} mount $ctr_id)
```

Comment:
Should probably use the `run` helper, set `root` to `$output`, and check `$status` here.